### PR TITLE
New icons

### PR DESCRIPTION
**Pull request by swese44**
# New icons from UX

## Pull request checklist

* [x] Component `README.md` file is up-to-date.
* [x] Component visual reference files are up-to-date.
* [x] Component is unit tested.

### CODE REVIEW

File: src/components/Icon/icons/MoreV.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+/* tslint:disable:max-line-length */
+// This file was auto-generated -- do not edit
+
+import * as React from 'react';
+
+const MoreV = function MoreV(props: any) {
```
Comment (unindented) [inline src/components/Icon/icons/MoreV.tsx]: Can't we name this better? `MoreVertical` or something?

File: src/components/Icon/icons/index.ts [hunk 2]
```diff
@@ -52,9 +54,11 @@ export default {
   mail,
   minus,
   more,
+  moreV,
```
Comment (unindented) [inline src/components/Icon/icons/index.ts]: This is not self-explanatory.

**Comment by unindented** [comment]
Why did the hovercard references change?

**Comment by swese44** [comment]
@mamuso made some minor updates to some icons, which caused a slight variation in the hovercard demos

File: src/components/Icon/icons/index.ts (hunk 2 above)
Comment (swese44) [reply src/components/Icon/icons/index.ts]: i asked @mamuso to rename it

**Comment by swese44** [comment]
if you use the image swipe you can see the different Mail icon

**Comment by unindented** [comment]
Ah right, some of the icons have changed in ways imperceptible to my developer eyes.

**Pull request merged**